src/app/components/Curation/CurationPromo/index.test.tsx

```diff
@@ -44,7 +44,7 @@ describe('Curation Promo', () => {
       <Fixture lazy={false} duration={123} type="video" />,
     );
 
-    const durationString = 'Duration, 2,03';
+    const durationString = ', Duration 2,03';
```

Comment:
Can we remove the leading `,` because the tests should work without it? (And is more readable)
```suggestion
    const durationString = 'Duration 2,03';
```

Reply:
Turns out the tests fail if we remove the comma, so best to leave as is!